### PR TITLE
Rails-React as one App: Setup

### PR DESCRIPTION
**Pull request by amare1990**
**In this project, I have:**

- [x] Followed Gitflow
- [x] Used `jsbundling-rails` gem to bundle both the rails and react as a single application.
- [x] Created API end point that generate a random greeting message in `json` format from the table having 5 message records.
- [x] Created redux store, action and reducers and a react Greeting component to display one of the five messages randomly when  
      the page is refreshed.
- [x] Fixed linters bugs.
- [x] Updated README.md template.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```
Comment (t-yanick) [inline config/routes.rb]: - [ ] Great job working on this project. Commented lines of code make our codebase look unprofessional. Kindly delete this line of commented code.

File: README.md [hunk 2]
```diff
@@ -80,209 +34,128 @@ After you're finished please remove all the comments and instructions!
...
+- rails-7.0.4.2
+- @reduxjs/toolkit
+- react-redux
+- react-dom
+- react-router-dom
+- @babel
 
 ### Setup
 
 Clone this repository to your desired folder:
```
Comment (t-yanick) [inline README.md:89]: - [ ] Kindly add in details the process a user needs to follow to be able to run your project locally. From the project requirements, this app is started with the command  `**(IMPORTANT: start the app with ./bin/dev command)**` which is absent from your ReadMe. Please make your ReadMe very professional

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @amare1990 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark: All required gems and dependencies used
:heavy_check_mark: API endpoint, routes, model, controller all added
:heavy_check_mark: Everything exists in application.js
:heavy_check_mark: The two apps working together

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (amare1990) [reply README.md:89]: Hi @t-yanick . Thank you for reviewing my project. The command `./bin/dev` is already there in the usage section. I think this command is used to run the project. Thanks

Comment (amare1990) [reply README.md:89]: Hi @t-yanick . Could you check the changes I made?

**Review by Owusu-Desmond — APPROVED** [review]
Hi @amare1990 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**